sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Infrastructure/MockEventProcessorClient.cs

```diff
@@ -0,0 +1,289 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Moq;
+using static Azure.Messaging.EventHubs.Tests.EventProcessorClientTests;
```

Comment:
Given that the `internal` types will be made `private`, we won't be needing this anymore. So we are safe to remove it.